be/.env.example

```diff
@@ -0,0 +1,2 @@
+DATABASE_URL=
+PORT=
```

Comment:
I think you should add comments on top of each env var explaining what they are for. Although these are quite simple it's a good practice.